workflow-controller/controller-service/src/main/java/io/hyscale/controller/activity/ValidatorActivity.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.controller.activity;
+
+import io.hyscale.commons.models.Activity;
+
+/**
+ * Activities list for validation
+ *
+ */
+public enum ValidatorActivity implements Activity {
+    VALIDATING_SERVICE_SPEC("Validating service spec(s) "),
+    VALIDATING_PROFILE("Validating profile(s) "),
+    VALIDATING_MAPPING("Validating Service and Profile mapping"),
+    VALIDATING_DOCKER("Validating docker "),
+    VALIDATING_CLUSTER("Validating cluster "),
```

Comment:
With respect to the discussion , activity at every validation can be too detailed. We can remove these I guess

Reply:
The discussion was to improve the error message shown by service spec schema failures. We can discuss on this.